Zukini.UI.Examples.Features/Steps/SmokeTestSteps.cs

```diff
@@ -81,5 +86,57 @@ public void ThenIShouldSeeThatTheTableTagIsSupportedForTheFollowing(Table table)
                 Assert.IsTrue(page.IsBrowserSupported(browserName), $"Expected browser {browserName} to be supported.");
             }
         }
+
+        [Given(@"I create a delayed button")]
+        public void GivenICreateADelayedButton()
+        {
+            string jsButton = "setTimeout( createButton, 2000 ); function createButton() { var button = document.createElement(\"button\"); button.innerHTML = \"I am button\"; document.getElementsByTagName(\"body\")[0].appendChild(button); }";
+            Browser.ExecuteScript(jsButton);
+        }
+
+        [Then(@"the delayed button should eventually exist")]
+        public void ThenTheDelayedButtonShouldEventuallyExist()
+        {
+            var buttons = Browser.FindAllXPath("//button");
+            Assert.IsFalse(buttons.Count() > 0, "Button should not have existed yet");
+
```

Comment:
This step is dependent on a timing of execution. We could probably forego the check that the button does not exist prior to the check that it does exist.